src/db/clients/postgresql.js

```diff
@@ -291,6 +291,8 @@ function _configDatabase(server, database) {
     config.port = server.config.localPort;
   }
 
+  if (server.config.ssl) config.ssl = server.config.ssl;
```

Comment:
Could just break the if statement into a new line with curly brackets? Just to keep the code with the same pattern.